### PR TITLE
Component/button

### PR DESCRIPTION
**Pull request by MarcelloAragoni**
**Trello task**: https://trello.com/c/DzVHcHwu/6-s

## Description
Created a component Button.

## PrintScreen
image

### CODE REVIEW

File: src/Components/Button/Button.jsx [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export default function Button({text, type}){
```
Comment (ralfting) [inline src/Components/Button/Button.jsx]:
```suggestion
export default function Button({ text, type }) {

```

File: src/Components/Button/Button.jsx [hunk 2]
```diff
@@ -0,0 +1,5 @@
+export default function Button({text, type}){
+    return(
+        <button className={`button button--${type}`}>{text}</button>
```
Comment (ralfting) [inline src/Components/Button/Button.jsx]: I think that here is missing the style too, since you're implementing an isolated component, I suggest you add the visual style for this component and then put in the screenshot section on description a print about what the component looks like.

**Comment by ralfting** [comment]
And don't forget to add an image to the description :D

File: src/Components/Button/button.scss [hunk 3]
```diff
@@ -0,0 +1,14 @@
+.button {
+  height: 40px;
+  border-width: 0px;
+  color: white;
+  font-weight: bold;
+  padding-inline: 30px;
+  justify-self: center;
+
+  &--news {
```
Comment (ralfting) [inline src/Components/Button/button.scss]:
```suggestion
  &--primary {

```
You can call this as the primary button style, inspired by https://getbootstrap.com/docs/5.1/components/buttons/#examples

File: src/Components/Input/Input.jsx [hunk 4]
```diff
@@ -0,0 +1,5 @@
+import "./input.scss";
```
Comment (ralfting) [inline src/Components/Input/Input.jsx]: Are you ou changing to use PascalCase here, is that the reason for this new file, correct?

Comment (MarcelloAragoni) [reply src/Components/Input/Input.jsx]: this input file, was brought by the fetch with the main branch, i forgot to do the force push to update this one.
and, can you give me an explanation on the "PascalCase"

Comment (ralfting) [reply src/Components/Input/Input.jsx]: But it's not supposed to appear here, because this is a diff between your branch to main, I think that in main branch you're using lower-case 🤔

Comment (ralfting) [reply src/Components/Input/Input.jsx]: About the pascal case.
Is the way that you will name stuff, check it out:

### TLDR;
- camelCase
- PascalCase
- snake_case
- kebab-case

for more information check this post: https://betterprogramming.pub/string-case-styles-camel-pascal-snake-and-kebab-case-981407998841

Comment (ralfting) [inline src/Components/Input/Input.jsx]: Change the file name from `src/Components/Input/Input.jsx` to `src/Components/Input/input.jsx`

**Review by ralfting — APPROVED** [review]
You rock 🚀
